modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/Table/index.tsx

```diff
@@ -114,7 +117,7 @@ const Table: React.FC<TableProps> = ({
 
 	return (
 		<>
-			<ClayTable borderless className="testray-table" hover>
+			<ClayTable borderless className="testray-table" hover responsive>
```

Comment:
Create a property to validate responsiveness as this change can impact all tables